include/mbedtls/mbedtls_config.h

```diff
@@ -1146,6 +1146,11 @@
  * \warning If building without MBEDTLS_MD_C, you must call psa_crypto_init()
  * before doing any PKCS#1 v2.1 operation.
  *
+ * \warning When building with MBEDTLS_MD_C, all hashes used with this
+ * need to be available a built-ins (that is, for SHA-256, MBEDTLS_SHA256_C,
```

Comment:
[_Typo_] Did you mean "available **as** built-ins"? Same goes for paragraphs below.